kinematics_base/include/moveit/kinematics_base/kinematics_base.h

```diff
@@ -56,6 +56,39 @@ class RobotState;
 namespace kinematics
 {
 
+/*
+ * @enum KinematicSearches
+ * @brief Flags for choosing the type discretization applied on the redundancy during a multiple ik solution search
+ */
+namespace KinematicSearches
+{
+  enum KinematicSearch
+  {
+    ONE = 1,
```

Comment:
I don't know that the meaning of these values is obvious.  Each enum value must be clearly defined so that plugin implementers know what they mean.